content/2019-12-26-newsletter-22.md

```diff
@@ -45,6 +45,8 @@ If you want to mention something in [the next newsletter], send us a pull reques
 
 - [@eldruin] released a platform-agnostic [driver for the PCA9685][pca9685-driver] PWM LED/Servo controller driver and published a [blog post][pca9685-blog-post] including a video of the device controlling RGB LEDs and several Servo motors simultaneously.
 
+- [@eldruin] released a platform-agnostic [driver for Vishay's VEML6030][veml6030-driver] ambient light sensor and published a [blog post][veml6030-blog-post] including pictures of the device in a PCB of a commercial product where the application is written in Rust.
```

Comment:
> including pictures of the device in a PCB of a commercial product where the application is written in Rust.

What is this part trying to convey? What's the added benefit of the picture? As a reader, it confused me.

Reply:
I intended it as motivational. I removed it now since it is not working.